src/Interpreters/Context.cpp

```diff
@@ -322,6 +323,8 @@ struct ContextSharedPart : boost::noncopyable
     std::optional<MergeTreeSettings> merge_tree_settings TSA_GUARDED_BY(mutex);   /// Settings of MergeTree* engines.
     std::optional<MergeTreeSettings> replicated_merge_tree_settings TSA_GUARDED_BY(mutex);   /// Settings of ReplicatedMergeTree* engines.
     std::atomic_size_t max_table_size_to_drop = 50000000000lu; /// Protects MergeTree tables from accidental DROP (50GB by default)
+    String get_client_http_header_forbidden_headers;
```

Comment:
why not have it as `std::unordered_set<String>` so you don't construct it on every call?

Reply:
@antonio2368 Updated